main-client.js

```diff
@@ -8,9 +8,22 @@ import { _ } from 'meteor/underscore';
 const defaultNetworkInterfaceConfig = {
   path: '/graphql',
   options: {},
-  useMeteorAccounts: true
+  useMeteorAccounts: true,
+  // Pass a meteor accounts from another connection
+  meteorAccounts: Accounts,
```

Comment:
What's the purpose of allowing providing an Accounts from another connection? Wouldn't its _storedLoginToken still use the same localstorage key, or does it namespace it?